### DISCUSSION TITLE
Life Cycle RKE2 versions

### DISCUSSION THREAD

**Discussion by frit0-rb** [rancher/rke2#5887] [General]
Good afternoon everyone,

I looking for where can I check the Life Cycle about any versions of RKE2.

Are there any URL to do it?

thanks in advance

**Comment by brandond** [comment]
We follow upstream Kubernetes patch releases. When there is a new patch release of Kubernetes, there will be a corresponding release of RKE2 within a few days. When a minor version of Kubernetes is end-of-life and no longer receives patch releases, that minor is end of life for RKE2 as well.

https://kubernetes.io/releases/patch-releases/

**Reply by frit0-rb** [reply]
Thanks @brandond
